### ISSUE TITLE
Support for previewing animations, even if no model exists

### ISSUE THREAD

**Issue by BreakinBenny** [craftablescience/VPKEdit#158]
### Describe Your Suggestion

Currently, VPKEdit does not let us view model animations whereas HLMV++ does. This is best demonstrated when you load up a .MDL file without .VTX files, which gives us the Error placeholder instead of drawing a skeleton and a list of animations.

But maybe that's not really the point of this application…?

**Comment by craftablescience** [comment]
I haven't added animation parsing to the MDL parser yet, so when I do I'll add that functionality to VPKEdit. Interesting to see VTX files aren't required, I didn't know that was a thing

**Comment by BreakinBenny** [comment]
> I haven't added animation parsing to the MDL parser yet, so when I do I'll add that functionality to VPKEdit. Interesting to see VTX files aren't required, I didn't know that was a thing

Yes, well for starters the stock HLMVs (including SFM's which I just tested, I'm sure it's the same with CS:GO's SDK) refuse to load models that lack vertices.

**Comment by craftablescience** [comment]
I think I am going to keep VTX files as a requirement, all Valve official models require them and I've never seen a use for a model without them

**Comment by BreakinBenny** [comment]
Kinda expected that, and HLMV++ already allows viewing models with skeleton animations, with or without vertices.
